pkg/servicecentral/it/client_manual_test.go

```diff
@@ -0,0 +1,69 @@
+package it
+
+import (
+	"encoding/json"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/atlassian/voyager/pkg/servicecentral"
+	"github.com/atlassian/voyager/pkg/util"
+	"github.com/atlassian/voyager/pkg/util/auth"
+	"github.com/atlassian/voyager/pkg/util/logz"
+	"github.com/atlassian/voyager/pkg/util/pkiutil"
+	"github.com/atlassian/voyager/pkg/util/testutil"
+	"github.com/stretchr/testify/require"
+	"k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+const (
+	defaultServiceCentralURL = "https://services.prod.atl-paas.net"
+	fakeTestUsername         = "fcobb"
+)
+
+// You can run this file from root using
+// `go test -v pkg/servicecentral/it/client_manual_test.go`
```

Comment:
"how to use go" should live somewhere else 😛 i.e. we don't have such instructions anywhere else, why here?